scripts/build_a_package.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script builds wheels for a single package when trigged from a push to
+# a tag as part of a GitHub workflow (See .github/publish-a-package.yml). The
+# wheel is then published to PyPI.
+
+set -ev
+
+if [ -z $GITHUB_REF ]; then
+  echo 'Failed to run script, missing workflow env variable GITHUB_REF'
+  exit -1
+fi
+
+PKG_NAME_AND_VERSION=${GITHUB_REF#refs/tags/*}
```

Comment:
One other bash nit, i'd use lowercase variable names for non-environment variables.

Reply:
That's a good idea! I was following `./scripts/build.sh` but we should probably update that one instead. Updated!